docs/source/cpp/compute.rst

```diff
@@ -199,6 +199,8 @@ the input to a single output value.
 +--------------------+-------+------------------+------------------------+----------------------------------+-------+
 | count              | Unary | Any              | Scalar Int64           | :struct:`CountOptions`           | \(2)  |
 +--------------------+-------+------------------+------------------------+----------------------------------+-------+
+| count_distinct     | Unary | Any              | Scalar Int64           | :struct:`CountOptions`           | \(2)  |
```

Comment:
Does this currently supported nested types?

Reply:
thanks!